### DISCUSSION TITLE
Image compress have a lot of noises

### DISCUSSION THREAD

**Discussion by jamedom** [upscayl/upscayl#530] [Ideas]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have checked that this issue isn't addressed in any other issue or pull request.

### Description

I would like you to improve compress quality if it's possible. 

### Related Problem(s)

After I convert my image using upscayl I always compress it in order to get a smaller yet without losing image quality. Meanwhile upscayl just add a new feature which allow me to compress image without using other app to help but I notice one problem is that it created a lot of noises in the image. While i test it with other app like Fileminimizer picture, it does a better job in compressing by making a smaller size image and have less noise. It would be better if you could improve it. Thank you.

### Alternatives

I try to compress with 30% 50% and 70% and use it to compare with image that I compress with Fileminimizer Picture which i use 85% compress. As the result, the 30% compress with upscayl do a better job but the file is too big while 50 and 70% have a lot of noise in it compared to the image that I compress with Fileminimizer Picture and the size is quiet similar.

**Comment by aaronliu0130** [comment]
Hmm, but that program is proprietary freeware and we can’t incorporate it here. An open source tool, preferably in JavaScript or typescript would be very helpful.
